package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@adobe/aio-lib-core-tvm",
   "version": "1.3.0",
   "description": "A JS client to access the Adobe I/O token vending machine",
-  "main": "index.js",
+  "main": "lib/TvmClient.js",
```

Comment:
This is not equivalent to the index.js and breaks the interface

=> index returns `require('lib/TvmClient.js').TvmClient` while this returns { TvmClient }

Reply:
Wouldn't the change made to `module.exports` in TvmClient.js take care of this ?

Reply:
yup I went too fast, thanks :)